packages/engine/Source/Shaders/Builtin/Functions/pbrNeutralTonemapping.glsl

```diff
@@ -0,0 +1,23 @@
+// KhronosGroup https://github.com/KhronosGroup/ToneMapping/tree/main/PBR_Neutral
+
+// Input color is non-negative and resides in the Linear Rec. 709 color space.
+// Output color is also Linear Rec. 709, but in the [0, 1] range.
+
+vec3 czm_pbrNeutralTonemapping( vec3 color ) {
+    const float startCompression = 0.8 - 0.04;
+    const float desaturation = 0.15;
+
+    float x = min(color.r, min(color.g, color.b));
+    float offset = x < 0.08 ? x - 6.25 * x * x : 0.04;
```

Comment:
We have a `czm_branchFreeTernary` operator that can improve performance. Generally the fewer control flow statements in a shader, the better.

Reply:
I just copied the glsl file directly from Khronos with no changes so thanks for calling these out